### PR TITLE
Add Most Vulnerable Children summary

### PR DESCRIPTION
**Pull request by samshara**

### CODE REVIEW

File: client/src/views/Dashboard/Summary/index.js [hunk 1]
```diff
@@ -218,6 +222,17 @@ export default class Summary extends PureComponent {
...
+    mostVulnerableChildrenParams = (key, data) => ({
+        title: data.label,
+        value: data.value,
+    });
+
+    mostVulnerableChildrenGroupParams = d => ({ children: d });
+
+    mostVulnerableKeySelector = d => d.label;
+
+    mostVulnerableGroupKeySelector = d => d.type;
```
Comment (tnagorra) [inline client/src/views/Dashboard/Summary/index.js]: These methods should either be static functions or outside the Class

File: client/src/views/Dashboard/Summary/index.js [hunk 2]
```diff
@@ -423,6 +438,204 @@ export default class Summary extends PureComponent {
...
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child with Disability',
+                value: data.totalChildWithDisability,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Belongs to a Marginalized Group',
+                value: data.totalChildBelongsToA_marginalizedGroup,
```
Comment (tnagorra) [inline client/src/views/Dashboard/Summary/index.js:603]: This doesn't look like a proper camelcase

File: client/src/views/Dashboard/Summary/index.js [hunk 3]
```diff
@@ -423,6 +438,204 @@ export default class Summary extends PureComponent {
...
+                type: 'Serious Discrimination',
+                label: 'Child Living in Isolation',
+                value: data.totalChildLivingInIsolation,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Total Serious Discrimination',
+                value: data.totalSeriousDiscimination,
+            },
+        ].filter(v => v.value);
```
Comment (tnagorra) [inline client/src/views/Dashboard/Summary/index.js:635]: It's better to filter numbers using `v => isDefined(v.value)` as `0` is a falsy value

File: client/src/views/Dashboard/Summary/index.js (hunk 2 above)
Comment (thenav56) [reply client/src/views/Dashboard/Summary/index.js:603]: This can be skiped.

File: client/src/views/Dashboard/Summary/index.js (hunk 3 above)
Comment (samshara) [reply client/src/views/Dashboard/Summary/index.js:635]: yeah 0 is also filtered

File: client/src/views/Dashboard/Summary/index.js (hunk 2 above)
Comment (samshara) [reply client/src/views/Dashboard/Summary/index.js:603]: it's from server

**Pull request merged**